bin/test.sh

```diff
@@ -8,10 +8,14 @@ set -eo pipefail
 export DEVELOPMENT=1
 export DJANGO_CONFIGURATION=Test
 
-# pass CI env vars into docker containers for codecov submission
-[ ! -z ${CI+check} ] && \
-    echo "Getting Codecov environment variables" && \
+# run docker compose with the given environment variables
+
+if [[ ! -z "${CI}" ]]; then
```

Comment:
FYI you could replace `! -z` with the equivalent `-n` here

Reply:
Ohh... I never get these right. I always just google stackoverflow and trial-and-error till it works. It always gets tricky when things are set but set to an empty string.